xsdata/formats/generators.py

```diff
@@ -38,17 +36,20 @@ def __init__(self):
             raise CodeGeneratorError("Missing generator templates directory")
 
         self.env = Environment(loader=FileSystemLoader(str(self.templates_dir)))
-        self.resolver = DependenciesResolver()
 
     def template(self, name: str) -> Template:
         return self.env.get_template("{}.jinja2".format(name))
 
     @abstractmethod
-    def render(
-        self, schema: Schema, classes: List[Class], package: str
-    ) -> Iterator[Tuple[Path, str, str]]:
+    def render(self, classes: List[Class]) -> Iterator[Tuple[Path, str, str]]:
         pass
 
+    def module_name(self, name: str) -> str:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5141157)